cmds/bmp/login_command.go

```diff
@@ -86,5 +87,6 @@ func (cmd loginCommand) Execute(args []string) (int, error) {
 		return 1, err
 	}
 
+	fmt.Println("Login Successfully!")
```

Comment:
I'd say this message should be: `"Login successful"` since `"Login Successfully"` is not correct English